app/components/views/Offset/index.tsx

```diff
@@ -599,9 +599,8 @@ export const Offset = (props: Props) => {
     .map((tkn) => ({
       ...tokenInfo[tkn],
       description: (function () {
-        if (isLoading) return <Trans id="shared.loading" />;
+        if (isLoading) return <Trans id="shared.loading">Loading...</Trans>;
         if (!props.isConnected || !Number(balances?.[tkn])) return "0";
-        return Number(balances?.[tkn]).toFixed(2);
```

Comment:
Why is this line removed here?

Reply:
Great pickup, I think it was from auto formatting 🤦‍♂️